lib/glimesh_web/live/user_live/components/streamer_title.ex

```diff
@@ -0,0 +1,90 @@
+defmodule GlimeshWeb.UserLive.Components.StreamerTitle do
+  use GlimeshWeb, :live_view
+
+  alias Glimesh.Presence
+  alias Glimesh.Streams.StreamMetadata
+
+  @impl true
+  def render(assigns) do
+    ~L"""
+    <%= if @user do %>
+      <%= unless @user.username == @streamer.username do %>
+        <h5 class=""><span class="badge badge-danger">Live!</span> <%= @title %></h5>
```

Comment:
I see this line twice, can we compact it or add a new flag to replace these two if statements?